package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "app",
+  "private": true,
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "@babel/core": "^7.23.9",
+    "@babel/preset-env": "^7.23.9",
+    "@babel/preset-react": "^7.23.3",
+    "@reduxjs/toolkit": "^2.2.1",
+    "babel-loader": "^9.1.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^9.1.0",
+    "react-router-dom": "^6.22.1",
+    "redux-logger": "^3.0.6"
+  },
+  "devDependencies": {
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
```

Comment:
- [ ] Dear **George**, you did awesome work and after some small hurdles I saw that your application is working perfectly, I mean your `API` greeting and your `UI` for greeting rendering are working fine, but you should make sure that if someone uses your application and tries to run it, he/she will face a bad user experience and definitely wouldn't show interest further to look at your application output or result. Typical case you can consider, hiring a manager, HR or any other developer. Please always ensure that your application is working fine, at the beginning, I got some errors related to missing `dependencies` or `devDependencies`. Like for example, you are using webpack but the `webpack` and `webpack-cli` libraries are missing in your `package.json` `dependency` section, whereas for API requests on front-end you are using `Axios`, that also missing from the `dependencies` object list of the `package.json` file. Could you add them all, and make sure that your application is working without having any issues?